R/score_untrained.R

```diff
@@ -0,0 +1,167 @@
+#' @title Calculate model importance for a single task when using an untrained
+#' ensemble
+#' @description
+#' Evaluate the importance of ensemble component models by quantifying their
+#' contribution to the prediction accuracy of an untrained ensemble for each
+#' combination of model task.
+#'
+#' @param single_task_data A data.frame with the predictions for a single
+#' forecast task in a model_out_tbl format. The data must contain only one
+#' `output_type`, which must be one of the following:
+#' `mean`, `median`, `quantile`, or `pmf`.
```

Comment:
consider adding sample as a valid output_type. Would mean you would have to align that output type with valid scores.